### ISSUE TITLE
💡 [REQUEST] - Allow user to chose position of pulley

### ISSUE THREAD

**Issue by TheSecondWing** [madmustachecompany/Blast-Workout-App#81]
### Summary

Hello,

For exercises like Overhead Triceps Extensions that can be down with a low or high pulley, we should be able to select the position of said pulley.

Otherwise, the only solution we have for such exercises is to leave a note. And also, because it's currently listed as the same exercise, both variations are gonna fill up the same stats charts.

Thanks.

**Comment by TheSecondWing** [comment]
Hi,

Correct me if I'm wrong but after further investigation, it seems like the **high pulley** Overhead Triceps Extension is actually listed as a variation under Skullcrusher; because in the French version there is a variation in Skullcrusher saying "Standing (bent over on a cable machine)". Am I right?

If not, under what exercise would you log the movement depicted in the picture attached?

Thanks!

![17221101-Cable-High-Pulley-Overhead-Tricep-Extension_Upper-Arms_max-353x150.jpg](https://github.com/madmustachecompany/Blast-Workout-App/assets/156791196/05575f3f-139d-4644-8def-a4006e0ea400)

**Comment by madmustachecompany** [comment]
@TheSecondWing,

You are absolutely right. The Skull Crusher exercise is perfectly suited to the high pulley triceps extension exercise performed with cable, and it's our recommended choice.

However, you do raise an interesting point as it's possible to perform the "skull crusher" movement on the cable machine, either lying down (on a bench with a low pulley) or standing up (with a high pulley). The custom exercise feature currently under development should resolve this issue.
